README.md

```diff
@@ -40,6 +40,12 @@ documentation at https://articlequality.readthedocs.io .
 2. install the package itself and its dependencies `python setup.py install`
 3. You can verify that your installation worked by running `make enwiki_models` to build the English Wikipedia article quality model or `make wikidatawiki_models` to build the item quality model for Wikidata
 
+### Retraining the models
+
+To retrain a model, run `make -B MODEL` i.e `make -B models/wikidatawiki.item_quality.gradient_boosting.model`.
```

Comment:
Also, I was wondering if we could instead pick the top-most target here: `wikidatawiki_models`? That refers internally to `models/wikidatawiki.item_quality.gradient_boosting.model`.

Reply:
it is an example. it can be anything.

Reply:
Yes, e.g. sounds better suited for the task.

Reply:
How about we give a bit more information:

```suggestion
To retrain a model, run `make -B MODEL` e.g. `make -B wikidatawiki_models`. This will redownload the labels, re-extract the features from the revisions, and then retrain and rescore the model.
```